PythonImportant/leap year dictator.py

```diff
@@ -0,0 +1,17 @@
+def is_leap(year):
```

Comment:
Only this file is required and not the one outside the directory `PythonImportant`.

Also please change the directory name to `Leap Year Detector`